rustfmt.toml

```diff
@@ -0,0 +1,3 @@
+group_imports = "StdExternalCrate"
+ignore = ["src/arrow2"]
```

Comment:
we should probably also ignore `src/parquet2` as that's also one that we forked

Reply:
Updated!